README.md

```diff
@@ -94,7 +94,15 @@ The app has the following features:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+## 🚀 Live Demo <a name="live-demo"></a>
+- Coming soon
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Project Presentation <a name="presentation"></a>
+- Click <a href="https://www.loom.com/share/1706468c115d406cb2b925df89fc3ba7?sid=26240e7e-77f7-4abe-ada3-39febc516083)">here</a> to see the presentation.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Great job making a video highlighting the features of your application. However, there's a requirement that is missing from the video. It is required that your video should include highlight about a piece of the code you find interesting or you're proud of making. That is not yet included in your video. Kindly make a new video that includes that requirement.

![piece-of-code](https://github.com/wacoo/smart-budget-ror-capstone/assets/26597432/fd7b2e35-02cf-42ae-b548-595e9ec7c40d)